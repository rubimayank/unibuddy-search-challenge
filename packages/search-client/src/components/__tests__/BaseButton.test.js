import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import BaseButton from '../BaseButton';

describe('BaseButton', () => {
  it('it should render', () => {
    const snap = renderer.create(<BaseButton />).toJSON();
    expect(snap).toMatchSnapshot();
  });
});
