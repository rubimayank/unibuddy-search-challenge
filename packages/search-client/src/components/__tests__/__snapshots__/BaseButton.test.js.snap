// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseButton it should render 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  overflow: hidden;
  padding: 8px 16px;
  margin: 2px;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: all 150ms linear;
  transition: all 150ms linear;
  text-align: center;
  white-space: nowrap;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: capitalize;
  background: none;
  border: 0 none;
  border-radius: 2px;
  font-size: 14px;
  line-height: 1.3;
  -webkit-appearance: none;
  -moz-appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  vertical-align: middle;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0:hover {
  -webkit-transition: all 150ms linear;
  transition: all 150ms linear;
  opacity: .85;
}

.c0:active {
  -webkit-transition: all 150ms linear;
  transition: all 150ms linear;
  opacity: .75;
}

.c0:focus {
  outline: none;
}

.c0:disabled {
  cursor: not-allowed;
  opacity: .50;
}

.c0:disabled:hover {
  opacity: .50;
}

<button
  className="c0"
/>
`;
